dotcom-rendering/src/components/TextBlockComponent.tsx

```diff
@@ -187,7 +187,7 @@ const styles = (format: ArticleFormat) => css`
 		}
 	}
 
-	li:before {
+	ul li:before {
```

Comment:
Is this meant to be `ul li` (any descendant) or `ul > li` (direct child)?

Reply:
Good point! I'll change it to `ul > li` because if someone puts an ordered list inside an unordered `li` we get:

* with `ul li`
 
![image](https://github.com/user-attachments/assets/8dcfb5d2-8037-4ea4-b5cb-b3ec7b4f7c27)


* with `ul > li`

![image](https://github.com/user-attachments/assets/71873e6d-859d-4508-b8b3-9be43a82f645)